src/sage/algebras/lie_algebras/free_lie_algebra.py

```diff
@@ -387,7 +389,7 @@ def _repr_(self):
             sage: LieAlgebra(QQ, 3, 'x')
             Free Lie algebra generated by (x0, x1, x2) over Rational Field
         """
-        n = tuple(map(LieGenerator, self._names, range(len(self._names)))) # To remove those stupid quote marks
+        n = tuple(map(LieGenerator, self._names, range(len(self._names))))  # To remove those stupid quote marks
```

Comment:
looks like this TODO could be removed?

Reply:
There is no TODO here, just a remark that this is a complicated but necessary way to get the correct string.

Reply:
Sorry, was reading too fast. Sorry for wasting your time.